fda/depth_measures.py

```diff
@@ -0,0 +1,173 @@
+"""Depth Measures Module.
+
+This module includes different methods to order functional data,
+from the center (larger values) outwards(smaller ones)."""
+
+import math
+import numpy as np
+from scipy.stats import rankdata
+from fda.grid import FDataGrid
```

Comment:
Please, use `.grid` instead of `fda.grid` if you can. It will be easier to change the package name that way.